packages/snage/src/config/load.ts

```diff
@@ -5,9 +5,56 @@ import {parseConfig} from './parser';
 import {Config, Field, hasProvided, RawConfig, RawField} from './type';
 import * as E from 'fp-ts/lib/Either';
 import * as A from 'fp-ts/lib/Array';
+import * as O from 'fp-ts/lib/Option';
+import * as IO from 'fp-ts/lib/IO';
+import * as IOE from 'fp-ts/lib/IOEither';
 import {pipe} from 'fp-ts/lib/pipeable';
 import {getValueProvider} from '../provider/provider';
 import {getOrdering} from '../query/sort';
+import {ConfigParameterName, printAndExit} from '../command/common';
+import {flow} from 'fp-ts/lib/function';
+
+export const getConfig = (): E.Either<string, Config> => pipe(getConfigFile()(), E.chain(loadConfig));
```

Comment:
Wenn nur eine operation im pipe gemacht wird könnte auch E.either.OP genutzt werden, hast du da ne preferenz, ich hab bisjetzt beides von dir gesehen :D.
```
export const getConfig = (): E.Either<string, Config> => E.either.chain(getConfigFile(), loadConfig);
```

Reply:
Ich würde sagen, was lesbarer aussieht. In diesem Fall finde ich deinen Vorschlag lesbarer.